src/data/constants/loadingTypes.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable import/prefer-default-export */
+export const assetLoading = {
+  LOCK: 'LOCK',
```

Comment:
Is this where other kinds of loading would go?

Like

```
export const tableLoading = {
   FILTER: 'FILTER',
   SORT: 'SORT',
   ...
}
```

and

```
export const assetLoading = {
   DELETE: 'DELETE'
   ...
}
```

?

Reply:
Yep! I'd like to see this become the home for that. What do you think?